.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build perlun.eu.org web site
+
+on: [push]
```

Comment:
This could be edited to only run on PRs, but OTOH it will be revisited soon when we convert this to `sitegen` anyway, so maybe it’ll do for now.